src-db/database/sourcedata/ETCOP_APP_TOOL.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <data>
-<!--FDAF15A61F114A8CA754DD4DC12BE2E1--><ETCOP_APP_TOOL>
```

Comment:
Is it okay to change the ID of a tool? Could it cause problems in environments that already had it installed?